cudf/io.py

```diff
@@ -42,6 +42,9 @@ def read_csv(filepath, lineterminator='\n',
     dtype : list of str or dict of {col: dtype}, default None
         List of data types in the same order of the column names
         or a dictionary with column_name:dtype (pandas style).
+    quoting : bool, default False
+        If False, any quotes will be removed from start & end of string fields
+        If True, any quotes on fields are included in nvstrings entries
```

Comment:
```suggestion
        If True, any quotes will be included in string fields
```
Suggest using parallel structure as shown between the False and True conditions to make behavior clearer. Also consider swapping the order of True and False.

Reply:
I'm trying, too char-count concisely (<80 chars per line), to convey that any quotes at the start and end of the field, _if they exist_, will be included in the returned nvstring array, but if the field wasn't wrapped in quotes, quotes will not be added. Will take a second stab at this docstring.

I don't want to call them "string fields" until we have deeper integration of nvstrings into cuDF.